BPRBlazor/Pages/Results.razor.cs

```diff
@@ -1,55 +1,20 @@
-﻿using BPR.Mediator.Models;
-using BPRBlazor.ViewModels;
+﻿using BPRBlazor.ViewModels;
 using Microsoft.AspNetCore.Components;
-using Microsoft.JSInterop;
 
 namespace BPRBlazor.Pages;
 
 public partial class Results : ComponentBase
 {
-    private ResultModel? _resultModel;
-    private List<ViolationModel> _filteredViolations = new();
-    
-    protected override async Task OnAfterRenderAsync (bool firstRender) 
-    {
-        if (firstRender)
-        {
-            _resultModel = new ResultModel
-            {
-                Violations = (await ProtectedLocalStore.GetAsync<List<ViolationModel>>("violations")).Value ?? new()
-            };
-            _filteredViolations = new List<ViolationModel>(_resultModel.Violations);
-            StateHasChanged();
-        }
-    } 
-    
-    private void HandleViolationType(ViolationTypeViewModel value)
-    {
-        if (value.IsChecked && _resultModel != null)
-        {
-            _filteredViolations.AddRange(_resultModel.Violations.Where(violation => violation.Type == value.ViolationType));
-        }
-        else
-        {
-            _filteredViolations.RemoveAll(violation => violation.Type == value.ViolationType);
-        }
-    }
+    private List<ResultViewModel> _results = new();
 
-    private List<ViolationTypeViewModel> GetCurrentViolationTypes()
+    protected override async Task OnInitializedAsync()
     {
-        if (_resultModel == null || !_resultModel.Violations.Any())
-        {
-            return new List<ViolationTypeViewModel>();
-        }
-        
-        return _resultModel.Violations.Select(violation => violation.Type)
-            .Distinct()
-            .Select(violation => new ViolationTypeViewModel(violation))
-            .ToList();
+        var resultModel = await ResultService.GetAllResultsAsync();
+        _results = Mapper.Map<List<ResultViewModel>>(resultModel);
     }
     
-    private async Task DownloadPdf()
```

Comment:
are we removing the download pdf version, or?

Reply:
We moved it in ResultsDetails